Tests/SphericalExtractionTest/SphericalExtractionTest.inputs

```diff
@@ -0,0 +1,33 @@
+verbosity = 0
+N1 = 32
+N2 = 32
+N3 = 32
+L = 16
+
+chk_prefix = TestChk_
+plot_prefix = TestPlt_
+
+num_points = 3
+
+num_ghosts = 3
+max_level = 1
+regrid_interval = 1 1 1 1 0 0 0 0 0
+ref_ratio = 2 2 2 2 2 2 2 2 2
```

Comment:
I guess you don't need this many `regrid_interval`s and `ref_ratios`s

Reply:
I guess not but it doesn't hurt in case someone wants to increase the number of levels in this test for whatever reason.